_layouts/news.html

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+---
+
+          
+<div class="news">
+    {% if site.news != blank -%} 
+    {%- assign news_size = site.news | size -%}
+    <div class="table-responsive" {% if site.news_scrollable and news_size > 3 %}style="max-height: 10vw"{% endif %}>
+      <table class="table table-sm table-borderless">
+      {%- assign news = site.news | reverse -%}
+      {% if site.news_limit %}
+      {% assign news_limit = site.news_limit %}
+      {% else %}
+      {% assign news_limit = news_size %}
+      {% endif %}
+      {% for item in news limit: news_limit %} 
+        <tr>
+          <th scope="row">{{ item.date | date: "%b %-d, %Y" }}</th>
+          <td>
+            {% if item.inline -%} 
+              {{ item.content | remove: '<p>' | remove: '</p>' | emojify }}
+            {%- else -%} 
+              <a class="news-title" href="{{ item.url | relative_url }}">{{ item.title }}</a>
+            {%- endif %} 
+          </td>
+        </tr>
+      {%- endfor %} 
+      </table>
+    </div>
+  {%- else -%} 
+    <p>No news so far...</p>
+  {%- endif %} 
+  </div>
```

Comment:
looks like this code is mostly a copy-paste from `_includes/news.html`. instead of adding this extra layout, let's just add `{% include news.html %}` directly in `_pages/news.md`.

Reply:
Yes, it is copy-paste except for the h2 header `news`. If that is not removed there will be two `news` headers.

Reply:
let's move the header to `_layouts/about.html` so that `_includes/news.html` can be included in `_pages/news.md` without creating the double header.